app/src/main/java/org/kiwix/kiwixmobile/library/LibraryAdapter.java

```diff
@@ -370,10 +380,24 @@ protected FilterResults performFiltering(CharSequence s) {
             .toList()
             .blockingGet();
 
-        listItems.add(new ListItem(context.getResources().getString(R.string.your_languages),
+        // Get all selected displayed languages from books.
+        HashSet<String> selectedLanguageSet = new HashSet<>();
+        for (Book book : selectedLanguages) {
+          selectedLanguageSet.add(bookUtils.getLanguage(book.getLanguage()));
+        }
+        
+        listItems.add(
```

Comment:
Did you refactored the code using Square codestyle?

Reply:
I just reformat the code.